### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by plopoyop**
## Problem
- app not up to date
- api access is broken
- bad permission on user data dir when adding a user
## Solution
- fix api access by improving the unprotected_uri setting
- fix a permission problem on hook
- upgrade to 1.4.3

## PR Status
- [X] Code finished.
- [ ] Tested with Package_check.
- [X] Fix or enhancement tested.
- [X] Upgrade from last version tested.
- [X] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/freshrss_ynh%20PR83%20(yalh76)/)

### CODE REVIEW

**Comment by Alkarex** [comment]
Fix https://github.com/YunoHost-Apps/freshrss_ynh/issues/81

**Comment by yalh76** [comment]
Thanks for the work. 

Can someone test the package ? 
- installation
- upgrade from previous version

File: scripts/upgrade [hunk 1]
```diff
@@ -155,6 +155,7 @@ ynh_replace_string "__APP__" $app ../conf/freshrss.cron
 cp ../conf/freshrss.cron "$cron_path"
 chmod 644 "$cron_path"
 
+rm /tmp/FreshRSS.log
```
[the remark this answers is not in the document]
Comment (plopoyop) [reply scripts/upgrade]: I may do something like :
```sh 
if [ -f /tmp/FreshRSS.log ]; then
	ynh_secure_remove "/tmp/FreshRSS.log"
fi;
```

**Comment by yalh76** [comment]
Last Package_check is ok. 

@plopoyop if you want to merge the pull request.

**Comment by Alkarex** [comment]
What is the oldest MySQL version you have to support? 
C.f. https://github.com/FreshRSS/FreshRSS/issues/2670

**Comment by yalh76** [comment]
> 
> 
> What is the oldest MySQL version you have to support?
> C.f. [FreshRSS/FreshRSS#2670](https://github.com/FreshRSS/FreshRSS/issues/2670)

I don't understand your question and the link with this PR.
Currently Yunohost is using 10.1.41-MariaDB-0+deb9u1

**Comment by Alkarex** [comment]
@yalh76 Ok, then no problem. We have just fixed a regression in FreshRSS for MySQL < 5.6 https://github.com/FreshRSS/FreshRSS/pull/2672 (will be released shortly in [FreshRSS 1.15.3](https://github.com/FreshRSS/FreshRSS/milestones/1.15.3) together with a couple of other fixes)

**Pull request merged**